pages/common/mmdc.md

```diff
@@ -0,0 +1,17 @@
+# mmdc
+
+> CLI for mermaid, a diagram generation tool with a domain-specific language.
+> A mermaid definition file is taken as input and a svg, png, or pdf file is generated as output.
```

Comment:
I think we are talking about formats, not the extension of the filename. So, I think `SVG`, `PNG` and `PDF` should be in upper case.